net-misc/streamlink/streamlink-0.1.0.ebuild

```diff
@@ -4,6 +4,7 @@
 
 EAPI=6
 
+PYTHON_REQ_USE='xml,threads'
```

Comment:
It is recommended to `(+)` those flags, for compatibility with other impls. See: https://wiki.gentoo.org/wiki/Project:Python/Implementation_USE_flags